docs/content.md

```diff
@@ -35,3 +35,29 @@ Component.extend( {
 	view: "<h1><content>Hi There!</content></h1>"
 } );
 ```
+
+The children of `<content>` can be used as a default value for rendering in the case that no content is passed.
```

Comment:
Instead of "no content is passed", I think we probably should introduce the LIGHT_DOM and SHADOW_DOM concepts?

Or for CanJS, maybe these should be `shadow view` and `light view`?